TUM Campus App/ViewControllerProvider.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ViewControllerProvider.swift
+//  TUM Campus App
+//
+//  Created by Robert on 06.10.17.
+//  Copyright © 2017 LS1 TUM. All rights reserved.
+//
+
+import UIKit
+
+class ViewControllerProvider {
```

Comment:
I don't know about this. Perhaps we should have a protocol for this and call it always directly on the class.
e.g.: `protocol InstantiableByStoryboard { static var shared: Self { get } }`

Don't worry too much about this. It's just a thought for later ;)